sample/README.md

```diff
@@ -0,0 +1,51 @@
+# Sample
+
+The Sample describes several demos of quickly spinning up different Prebid Server instances with various preset configurations. These samples are intended for audiences with little knowledge about Prebid Server and plan to play around with it locally and see how it works.
+
+# Installation
+
+In the Sample, we use `docker` and `docker-compose` to instantiate examples; with docker providing a unified setup and interface,  you can spin up a demo server instance locally with only one command without knowing all the complexities.
+The docker image used in `docker-compose.yml` is the `Dockerfile` residing in the root level of the repository. 
+
+## Option 1 - Standard Docker Engine
+Install `docker` and `docker-compose` via the [official docker page](https://docs.docker.com/compose/install/#scenario-one-install-docker-desktop). If you cannot use the official docker engine due to restrictions of its license, see the option below about using Podman instead of Docker. 
```

Comment:
In this PR's review comments it is mentioned that the commands below were useful to make it work without `podman`. Can we mention said commands here under option 1?
```
$ cd sample
$ docker compose build --no-cache //new command
$ docker-compose up <number>_<name>
```
I believe it makes for a more straightforward read to have the commands for option 1 (just `docker-compose`) separate from option 2 (`podman` + `docker-compose`)

Reply:
ok, sure i can separate them